model/ExpandedLicensing/Classes/NoAssertionLicense.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# NoAssertionLicense
+
+## Summary
+
+A concrete subclass of AnyLicenseInfo used by the NoAssertionLicense Individual in the ExpandedLicensing profile.
+
+
+## Description
+
+NoAssertionLicense should be used if the SPDX creator has attempted to but cannot reach a reasonable objective determination;
+the SPDX creator has made no attempt to determine this field; or
+the SPDX creator has intentionally provided no information (no meaning should be implied by doing so).
+
+## Metadata
+
+- name: NoAssertionLicense
+- SubclassOf: /SimpleLicensing/AnyLicenseInfo
```

Comment:
Why is [model/ExpandedLicensing/Classes/NoAssertionLicense.md](https://github.com/spdx/spdx-3-model/pull/588/files#diff-2ac21c661dbfd86dbf49893a827f500545da2efb57b35a664a69cb3ba09ac2b7) not  a subclass of [model/ExpandedLicensing/Classes/IndividualLicensingInfo.md](https://github.com/spdx/spdx-3-model/pull/588/files#diff-aadc6c742d7ee00d8904c325c153391f148e9cf32f56a5c1340da2f029099929), which is "A concrete subclass of AnyLicenseInfo used by Individuals in the ExpandedLicensing profile."

Reply:
This file should be deleted.